.spelling

```diff
@@ -529,4 +529,365 @@ ch6
+13
+21
+buildability
+ - 4-software-development-practices/4.3-version-control.md
+codebase
+ - 4-software-development-practices/4.3.2-git.md
+subfolders
+rebase
+facto
+ - 4-software-development-practices/4.4-pairprogramming.md
+pre-packaged
+ - 4-software-development-practices/4.5.1-unit-testing.md
+tl
+ - 4-software-development-practices/4.5.2-functional-testing.md
+programmatically
+WebDriver
+geckodriver
+ - 4-software-development-practices/4.5.3-code-styling.md
+snake_case
+camelCase
+formatters
+ - 4-software-development-practices/4.5.6-sonarqube.md
+http
+ - 5-release-management/5.2-declarative-pipelines.md
+everytime
+ - 5-release-management/5.3.1-maven.md
+reusability
+lifecycle
+groupId
+artifactId
+ - 5-release-management/5.3.2-maven-integration.md
+Axel
+Fontaine
+joda-time-hibernate
+buildnumber-maven-plugin
+buildnumber
+ - 5-release-management/5.3.3-make.md
+Makefile
+Makefiles
+ - 5-release-management/5.3.4-npm.md
+versioning
+ - 5-release-management/5.4-containers.md
+cpu
+ - 5-release-management/5.4.1-docker.md
+defacto
+ - 6-infrastructure-configuration-management/6.1-terraform.md
+APIs
+SDK
+availability_zone
+instance_type
+key_name
+security_groups
+parameterize
+staging.tfvars
+ - 6-infrastructure-configuration-management/6.1.4-terraform-providers.md
+declaratively
+templated
+makefile
+ReadMe
+reate
+dev
+devops
+ead
+pdate
+elete
+Structs
+structs
+Terraforms
+struct
+provider.go
+main.go
+datasources
+datasource
+schemas
+ - 6-infrastructure-configuration-management/6.2-ansible.md
+ansible
+httpd
+Ansible.com
+ - 6-infrastructure-configuration-management/6.3-chef.md
+Repos
+ - 7-kubernetes-container-orchestration/7.1.1-getting-started.md
+jsonpath
+ - 7-kubernetes-container-orchestration/7.1.2-app-management.md
+autoscale
+stateful
+replicaset
+unschedulable
+subcommand
+nginx
+kubectl
+ - 7-kubernetes-container-orchestration/7.1.4-cluster-management.md
+emptyDir
+uncordon
+schedulable
+ - 7-kubernetes-container-orchestration/7.1.5-kubectl-settings-and-usage.md
+kubeconfig
+subcommands
+ - 7-kubernetes-container-orchestration/7.2-volumes.md
+NFS
+persistentVolumes
+StorageClass
+StorageClasses
+pre-create
+volumeBindingMode
+ - 7-kubernetes-container-orchestration/7.3-probes.md
+Misconfigured
+liveness
+initialDelaySeconds
+periodSeconds
+timeoutSeconds
+successThreshold
+failureThreshold
+matchLabels
+ - 7-kubernetes-container-orchestration/7.4-rbac.md
+kubelets
+ClusterRoles
+RoleBindings
+healthz
+jenkins
+fluentd
+ - 7-kubernetes-container-orchestration/7.5-hpas.md
+tty
+busybox
+apiVersion
+ - 7-kubernetes-container-orchestration/7.6-webhooks.md
+apiGroups
+ - 7-kubernetes-container-orchestration/7.7-k8sthw.md
+PodSpecs
+cfssl
+readme
+ - 8-addendum/bitbucket.md
+Uncompress
+Jenkins_IP
+ - 8-addendum/ephemeral-jenkins.md
+job-dsl
+JCasC
+windspeed
+ - 8-addendum/nexus.md
+executables
+ - README.md
+deliverables
+bootcamper
+overcommunicate
+undercommunicate
+Docsify
+docsify
+H3
+H2
+ - _sidebar.md
+checkbox
```

Comment:
I already hate this spellcheck linter. Mostly because this file is impossible to digest easily. A quick look at spell checkers now points me to [CSpell](https://github.com/streetsidesoftware/cspell) which looks well maintained and extensible. Mostly griping here.